requirements.txt

```diff
@@ -1,2 +1,2 @@
-ansible>=2.4.2
+ansible==2.4.2
```

Comment:
Looks like 2.4.5 is there https://github.com/ansible/ansible/blob/stable-2.4/CHANGELOG.md.
So shouldn't we try something like > 2.4 and < 2.5?

Reply:
or just `ansible==2.4` to get the last one maybe

Reply:
:+1: to somehow pinning to "latest 2.4"

Reply:
the correct syntax seems to be : `ansible~=2.4`

`ansible==2.4` will give you ansible version `2.4.0` even if `2.4.5` is available